AmazonEmailService.pas

```diff
@@ -42,9 +41,9 @@ implementation
   AmazonEmailServiceRegions,
   BuildQueryParameters;
 
-constructor TAmazonEmailService.Create(const AWSRegion: AwsRegions; const AWSAccessKey, AWSSecretKey: string);
+constructor TAmazonEmailService.Create(const AWSRegion, AWSAccessKey, AWSSecretKey: string);
 begin
-  FAWSRegion := AWSRegion;
+  FAWSRegion := TAmazonEmailServiceRegions.DefineServiceURL(AWSRegion);
```

Comment:
If I got this correctly, this should be called `SESEndpointURL` or just `EndpointURL` and not region anymore, as this config is the Endpoint URL